packages/cfpb-design-system/src/components/cfpb-tooltips/tooltip.scss

```diff
@@ -0,0 +1,27 @@
+@use 'sass:math';
+@use '@cfpb/cfpb-design-system/src/abstracts' as *;
+@import 'tippy.js/dist/tippy.css';
+@import 'tippy.js/dist/border.css';
```

Comment:
Does it mess things up to use `@use` here?

Reply:
Nah, great idea.